### ISSUE TITLE
[Feature Request] Filters below Photos search

### ISSUE THREAD

**Issue by pablo03v** [cryptee/web-client#177]
**Is your feature request related to a problem? Please describe.**
There is no way to see photos that aren't tagged.

**Describe the solution you'd like**
When clicking in the search input, different filters should appear.  Example:
tags, description, time, album

**Additional context**
I am currently tagging all my photos and would appreciate a view where the photos that aren't yet tagged are displayed

**Comment by pablo03v** [comment]
Additionally to pressing enter to submit changes for album names, I would also love pressing enter when submitting tag changes instead of pressing a button

**Comment by johnozbay** [comment]
Thanks for this @pabloscloud! There's some really great ideas here! Part of why I didn't get back to this until now is because we had to get this week's update out before we could focus on feature additions across services. (the update had to do with service-startup / module-initialization etc to speed things up) 

We're making a list of filtering options and hoping to add them with the upcoming update!

Thanks again! 🙏🏻

Best,

J

**Comment by pablo03v** [comment]
Yeah, super hyped for anything that's about to come. No idea of what the 3 new things could be tbh but I'm totally into cryptees design as well as privacy/security decisions ✌️💛

**Comment by johnozbay** [comment]
Thank you so much for your kind comments, amazing feedback, thoughtful responses and feature suggestions! 🙏🏻 
Deeply appreciate you! The world needs more amazing people like you! ✌🏻
